src/satipc.c

```diff
@@ -134,8 +134,8 @@ int satipc_reply(sockets *s)
 	__attribute__((unused)) int rv;
 	get_ad_and_sipr(s->sid, 1);
 	s->rlen = 0;
-	LOG("satipc_reply (sock %d) handle %d, adapter %d:\n%s", s->id, s->sock,
-		s->sid, s->buf);
+	LOG("satipc_reply (sock %d) handle %d, adapter %d :", s->id, s->sock, s->sid);
+	LOGM("%s", s->buf);
```

Comment:
Hey, can we leave those things still at LOG ?
Next time we need to investigate I need to enable verbose for this module too...

Same for the next logm.

Otherwise, LGTM.

Reply:
> Hey, can we leave those things still at LOG ?
> Next time we need to investigate I need to enable verbose for this module too...

The idea is just the opposite! Include the full RTSP commands between the minisatip process and the SAT>IP server when the **general** log is enabled is annoying. The objective needs to be to enable them **only** when the debug of this module is enabled. So, when you enable the log of other modules you don't populate the log with irrelevant lines. Note that the line printing the _send_ of the message and the _receive_ of the response is every time included. So _only_ when you require it you can enable it (with a simple `-l satipc`).

I hate extensive logs!